esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,21 +3,27 @@
     UTC mode = True
 
 [task parameters]
-    metric = radiation_budget
+    fast = radiation_budget
+    medium = ensclus, heatwaves_coldwaves
 
 [scheduling]
+   cycling mode = integer
+   initial cycle point = 1
+   final cycle point = 5
     [[graph]]
-        R1 = install_cold => configure => process<metric> => compare<metric>
+        R1 = install_cold => configure => process<fast> & process<medium> => compare<fast> & compare<medium>
+        3/P2 = compare<fast>[-P2] => configure => process<fast> => compare<fast>
+        4/P3 = compare<medium>[-P3] => configure => process<medium> => compare<medium>
```

Comment:
`compare<fast>` must only rely on the success of `process<fast>` and should not include `process<medium>`.